metadata.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
+kind: Metadata
+releaseSeries:
+- major: 1
+  minor: 9
```

Comment:
We don't have v1.9 yet. Why is it here?

Reply:
I copied this straight as is in [CAMP3](https://github.com/metal3-io/cluster-api-provider-metal3/blob/main/metadata.yaml). CAPI also has one minor release more than is the current in their [metadata.yaml](https://github.com/kubernetes-sigs/cluster-api/blob/main/metadata.yaml). Also in this PR in metal-dev-env I create a local override layer that is one minor release higher that the current actual release.